### DISCUSSION TITLE
Blank popup on Test Request button

### DISCUSSION THREAD

**Discussion by testmabnanet** [scalar/scalar#3738] [Q&A]
Hi, 
I get a blank popup when I press Test Request button.
(with .net 8 and swagger)

**Comment by xC0dex** [comment]
Hey, does the browser console show any error message?

**Reply by testmabnanet** [reply]
![2024-11-04 12 18 34](https://github.com/user-attachments/assets/8f20d651-fc98-4e56-a4b8-0d8b62a2c48b)

Some warnings but no js error 

for each api I get a "Schema Error" scalar.js:3128 

{
  "_errors": [],
  "parameters": {
    "15": {
      "_errors": [],
      "examples": {
        "_errors": [],
        "default": {
          "_errors": [
            "Expected object, received string"
          ]
        }
      }
    },
    "16": {
      "_errors": [],
      "examples": {
        "_errors": [],
        "default": {
          "_errors": [
            "Expected object, received string"
          ]
        }
      }
    },
    "_errors": []
  }
}

**Reply by amritk** [reply]
Hello @testmabnanet it looks like the parameters may not be set correctly, could you link that section of the openapi.json?

**Comment by testmabnanet** [comment]
Hi, The documentation (for .net8)  did not say anything about openapi.json so I don't have  such file.

**Reply by xC0dex** [reply]
The `openapi.json` document is generated by the service dynamically. If you use `Swashbuckle` the URL of the generated document should be `/swagger/v1/swagger.json`. `v1` is the name of the document.

**Comment by xC0dex** [comment]
Hey @testmabnanet,

I think this should be fixed now. The reason this randomly occurs was a concurrency problem in the preview packages of Microsoft. With the official .NET 9 release, it’s fixed!
